.github/workflows/autopublish.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on:
+  pull_request: {}
+  push:
+    branches: [main]
+
+jobs:
+  main:
+    name: Build, Validate, and Publish
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: w3c/spec-prod@v2
+        with:
+          GH_PAGES_BRANCH: gh-pages
```

Comment:
```suggestion
          TOOLCHAIN: bikeshed
          GH_PAGES_BRANCH: gh-pages
          SOURCE: index.src.html
```

Reply:
If we’re not using a non-default SOURCE value, do we actually need TOOLCHAIN?